static/scripts/onboarding/onboarding.ts

```diff
@@ -552,12 +471,9 @@ const step2Handler = async () => {
 };
 
 const init = async () => {
-  let conf = await getConf(true);
-  if (conf !== undefined) {
+  if (defaultConf !== undefined) {
```

Comment:
As far as I understand we don't need the `getConf()` [method](https://github.com/ubiquity/pay.ubq.fi/blob/ef0b7b3f3858259a52beb92d6245fe13b1e2d71f/static/scripts/onboarding/onboarding.ts#L66) anymore (which fetches the config via an HTTP request) because the config now resides in github submodule. Hence we're free to delete the `getConf()` method along with unused vars like `DEFAULT_ORG, DEFAULT_REPO, DEFAULT_PATH`.

Reply:
We still need the `getConf` to merge the read config from the Ubiuibot repo. However I do agree we can delete the unused constants, will do.